manifests/g/GnuWin32/Gperf/3.0.1/GnuWin32.Gperf.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: GnuWin32.Gperf
+PackageVersion: 3.0.1
+PackageLocale: en-US
+Publisher: GnuWin32
+PackageName: 'GnuWin32: gperf'
```

```suggestion
PackageName: 'GnuWin32: Gperf'
```
to match Control Panel.